packages/clerk-js/src/core/clerk.ts

```diff
@@ -1077,6 +1052,34 @@ export default class Clerk implements ClerkInterface {
     }
   };
 
+  #setIntermediateState = () => {
```

Comment:
I was thinking about this a bit more and I'd like to coin the term `Transitive` state. 

`setTransitiveState`. In essence, this state helps us transition from a signed-out to a signed-in state and vice-versa.

WDYT?

Reply:
Transitive makes sense to me 👍🏻